x/clerk/keeper/common_test.go

```diff
@@ -0,0 +1,45 @@
+package keeper_test
```

Comment:
We can deduplicate the code in common_test inside clerk/keeper/ and clerk/
@saikumar0x1967 has done it in staking and checkpoint. Let's follow that.